pallets/funding/src/tests/2_evaluation.rs

```diff
@@ -76,6 +76,158 @@ mod round_flow {
 			let project_status = inst.get_project_details(project_id).status;
 			assert_eq!(project_status, ProjectStatus::FundingFailed);
 		}
+
+		#[test]
+		fn different_decimals_ct_still_works_as_expected() {
+			let mut inst = MockInstantiator::new(Some(RefCell::new(new_test_ext())));
+			let ed = MockInstantiator::get_ed();
+			let default_project_metadata = default_project_metadata(ISSUER_1);
+			let original_decimal_aware_price = default_project_metadata.minimum_price;
+			let original_price = <TestRuntime as Config>::PriceProvider::convert_back_to_normal_price(
+				original_decimal_aware_price,
+				USD_DECIMALS,
+				default_project_metadata.token_information.decimals,
+			)
+			.unwrap();
+
+			// Right PLMC conversion is stored. We move comma 4 places to the left since PLMC has 4 more decimals than USD.
+			let stored_plmc_price =
+				inst.execute(|| <TestRuntime as Config>::PriceProvider::get_price(PLMC_FOREIGN_ID).unwrap());
+			assert_eq!(stored_plmc_price, FixedU128::from_float(8.4));
+			let usable_plmc_price = inst.execute(|| {
+				<TestRuntime as Config>::PriceProvider::get_decimals_aware_price(
+					PLMC_FOREIGN_ID,
+					USD_DECIMALS,
+					PLMC_DECIMALS,
+				)
+				.unwrap()
+			});
+			assert_eq!(usable_plmc_price, FixedU128::from_float(0.00084));
+
+			let evaluation_threshold = inst.execute(|| <TestRuntime as Config>::EvaluationSuccessThreshold::get());
+
+			let min_evaluation_amount_usd = <TestRuntime as Config>::MinUsdPerEvaluation::get();
+			let min_evaluation_amount_plmc =
+				usable_plmc_price.reciprocal().unwrap().checked_mul_int(min_evaluation_amount_usd).unwrap();
+
+			// Check the evaluation threshold is calculated correctly for 2 decimals less than USD
+			let mut project_metadata_4_decimals = default_project_metadata.clone();
+			project_metadata_4_decimals.token_information.decimals = 4;
```

Comment:
Might be better to use extreme scenario, so decimals = 1? Or even 0 (not sure if allowed)?

Reply:
I'll add tests for both 0 and 1 decimal to be safe